libraries/mpxe-store/src/store.c

```diff
@@ -0,0 +1,174 @@
+#include "store.h"
```

Comment:
I feel like MPXE + in particular the communication between python and store.c deserves one of those big writeups on Confluence about how exactly all of it is implemented when you get time

Reply:
when I get time indeed